processing/src/main/java/io/druid/query/Druids.java

```diff
@@ -1134,6 +1136,12 @@ public SelectQueryBuilder intervals(List<Interval> l)
       return this;
     }
 
+    public SelectQueryBuilder descending(boolean descending)
```

Comment:
I see other methods in this class violate "normal" bean naming conventions. Oh well...